client/my-sites/plugins/plugin-details-header/test/index.jsx

```diff
@@ -11,7 +11,28 @@ jest.mock( 'react-redux', () => ( {
 } ) );
 
 jest.mock( '@automattic/calypso-config', () => {
-	const fn = jest.fn();
+	const fn = ( key ) => {
+		if ( 'magnificent_non_en_locales' === key ) {
```

Comment:
The main reason is assuring consistency with the path [localization functionality](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/themes/helpers.js#L55) on /themes which supports mag 16. 
I needed to mock this because the newly added `localizePath` was breaking the component tests.